lib/sprig/seed/attribute.rb

```diff
@@ -54,7 +54,7 @@ def computed_value?(value)
       end
 
       def computed_value_regex
-        /<%[=]?(.*)%>/
+        /(<%[=]?(.*?)%>)/
```

Comment:
that non-greedy thing is hella clever. aka i don't get it but i see that it works.